install_scripts/templates/kubernetes/init.sh

```diff
@@ -339,12 +339,14 @@ rookDeploy() {
     # never upgrade an existing rook cluster
     if k8sNamespaceExists rook && k8sNamespaceExists rook-system ; then
         logSuccess "Rook 0.7.1 already deployed"
+        maybeDefaultRookStorageClass
```

Comment:
Isn't the storage class always included in the yaml when you render replicated so the annotation will be applied with the rest of the yaml?

Reply:
yes the annotation will be applied with the rest of the yaml but this was intended for upgrade scenarios from an existing installations where the rook storageClass exists and needs to have the default annotation applied to it
let me know if i'm understanding your question incorrectly..

Reply:
when you upgrade you re-generate all the replicated yaml and apply it and the newly generated storage class should be in there and have the annotation. If so, there's no need to do it procedurally here.